### ISSUE TITLE
Accessibility audit: Sidebar toggle labeling 

### ISSUE THREAD

**Issue by cbeer** [ProjectMirador/mirador#3117]
> The German-language button labels are partly not meaningful. The "button" to fold and unfold the sidebar ("burger menu") has the label "Switch sidebar". It would be more meaningful if the button was labeled "Show sidebar" or "Hide sidebar" depending on its state (folded in or out). It is not necessary to differentiate between the states using the label if the unfolded state is specified using an aria- expanded attribute (see PS 4.1.2a). In this case the label could be reduced to "Sidebar".

**Comment by cbeer** [comment]
Perhaps related?

> 
The inscription of the button "Close auxiliary window for canvas navigation" in the sidebar of the digital copies is also misleading, as the index window is not closed when the button is activated. In this case, a caption like "Index" would be sufficient.

**Comment by jvine** [comment]
We discussed at the time whether "Show/Hide sidebar" would be better than "Toggle sidebar" - and based the choice on existing patterns elsewhere, including material.io.  

We do need to add `aria-expanded="true/false"` to the `<button>` to indicate the current state of the sidebar. English text can remain as-is. (I disagree that it should be just "Sidebar" - I don't think that's sufficient for the tooltip.)

The second issue ("Close auxiliary window for canvas navigation") is purely a translation error.
```
    "openCompanionWindow_canvas": "Index",
    "openCompanionWindow_canvas": "Hilfsfenster für die Leinwandnavigation schließen",
```
